### DISCUSSION TITLE
Hexo 文章管理：子文件夹管理 + 保留文章永久链接 - Muxiner's Blog

### DISCUSSION THREAD

**Discussion by giscus** [Muxiner/muxiner.github.io#32] [博客文章评论]
# Hexo 文章管理：子文件夹管理 + 保留文章永久链接 - Muxiner's Blog

使用年份子文件夹管理 Hexo 文章，并保留文章的永久链接（之前的链接）。

https://muxiner.github.io/organize-files/

**Comment by KeilAwk** [comment]
好东西捏  ο(=•ω＜=)ρ⌒☆
看大大评论区空荡荡滴就留了个评论在这里~
